### PR TITLE
Newseed Stage 1

### PR DESCRIPTION
**Pull request by Peartes**
[x] Stage 1 layout
[x] Basic validation 
[o] Still need to include functionality for adding more rows

### CODE REVIEW

File: src/newSeed/baseStage.scss [hunk 1]
```diff
@@ -0,0 +1,223 @@
+@import "styles/colors";
+// This styles are common to all the stages components
+
+.page.stageContainer {
+  &.stage1 {
+    .infoContainer {
+      background-color: $color-black-2;
+      padding: 2em;
+      border-radius: 15px;
```
Comment (dkent600) [inline src/newSeed/baseStage.scss:9]: em?

File: src/newSeed/stage1.ts [hunk 2]
```diff
@@ -1,5 +1,33 @@
...
+    } else if (!Utils.isValidUrl(this.seedConfig?.general?.projectWebsite)) {
+      message = "Please enter a valid website url";
+    }
+    else if (!this.seedConfig?.general?.category) {
+      message = "Enter a category";
+    } else if (!this.seedConfig?.general?.whitepaper) {
+      message = "We need your whitepaper address";
+    } else if (!Utils.isValidUrl(this.seedConfig?.general?.whitepaper)) {
+      message = "Please enter a valid website url";
+    } else if (!this.seedConfig?.general?.github) {
```
Comment (dkent600) [inline src/newSeed/stage1.ts:22]: delete this check and replace by padding `false` as the 2nd param to isValidUrl

**Review by dkent600 — CHANGES_REQUESTED** [review]
Can you please remove the changes to stages > 1 from this PR?  I would prefer to review them one at a time.

File: src/newSeed/baseStage.scss (hunk 1 above)
Comment (Peartes) [reply src/newSeed/baseStage.scss:9]: Yes 😂 to resize with font size

File: src/newSeed/stage1.ts (hunk 2 above)
Comment (Peartes) [reply src/newSeed/stage1.ts:22]: Delete the check ? or just add a false as a second param ?

File: src/newSeed/baseStage.scss (hunk 1 above)
Comment (Peartes) [reply src/newSeed/baseStage.scss:9]: It allows for better responsiveness and i investigated that

**Comment by Peartes** [comment]
I made the changes so you can review stage 1 independently @dkent600

File: src/newSeed/baseStage.scss (hunk 1 above)
Comment (dkent600) [reply src/newSeed/baseStage.scss:9]: the comment is on a line that is using px

File: src/newSeed/stage1.ts (hunk 2 above)
Comment (dkent600) [reply src/newSeed/stage1.ts:22]: Delete the check.  note what false does

Comment (dkent600) [reply src/newSeed/stage1.ts:22]: don't forget to update the error message

File: src/newSeed/baseStage.scss (hunk 1 above)
Comment (Peartes) [reply src/newSeed/baseStage.scss:9]: Aha, for border-radius

**Pull request closed without merging**